code/Model/EditableFormField/EditableFileField.php

```diff
@@ -194,7 +194,18 @@ public function validate()
         return $result;
     }
 
+    public function getFolderExists(): bool
+    {
+        return $this->Folder()->ID !== 0 ? true : false;
+    }
 
+    public function setProtectedFolder(): void
```

Comment:
```suggestion
    public function createProtectedFolder(): void
```
Apologies, just realised this isn't really a `set` method since we're not passing in a parameter